managed/services/management/backup/backups_service.go

```diff
@@ -429,19 +429,12 @@ func (s *BackupsService) GetLogs(ctx context.Context, req *backuppb.GetLogsReque
 			models.MongoDBRestoreBackupJob,
 		},
 	}
-
 	if req.ArtifactId != "" {
-		// this allows us to maintain backward compatibility.
 		jobsFilter.ArtifactID = req.ArtifactId
+	} else if req.RestoreId != "" {
+		jobsFilter.RestoreID = req.RestoreId
```

Comment:
What if both artifact ID and restore ID set? Or none of them?

Reply:
If none is set, we throw an error (in the else case). If both are set, we default to only using the artifact ID - though I guess we can send an error in that case as well.